src/pages/difficulty.js

```diff
@@ -0,0 +1,93 @@
+import * as React from 'react';
+import { Link } from 'gatsby';
+import '../styles/style.scss';
+
+// data
+const data = [
```

Comment:
I think long term, these data constants should be moved to a single place to not pollute the page components, so I’d recommend naming these a bit more descriptively (ie. `const DC_DESCRIPTIONS = …`) so when they’re moved we don’t have to hassle with their names later.

Reply:
I agree. I was working on breaking them out for a separate PR. Mostly was aiming to get this page working and on the board, so to speak.
Would you rather I finished breaking out the data constants and add it to this PR?

Reply:
it's fine to leave them here for now, but rename the variable names if you're already changing the PR